### PR TITLE
Improved help menu

### PR DESCRIPTION
**Pull request by technetos**
This PR adds a new help menu to the bot along with `?help command` commands that bring up that commands help menu.  

image

#55 #14

### CODE REVIEW

**Comment by tinaun** [comment]
i wonder if the help message can show what commands are only available for mods (ban / coc etc)

**Comment by technetos** [comment]
> i wonder if the help message can show what commands are only available for mods (ban / coc etc)

Yep! #57

File: src/api.rs [hunk 1]
```diff
@@ -75,6 +75,31 @@ pub(crate) fn slow_mode(args: Args) -> Result<()> {
...
+```
+will set slowmode on the `#bot-usage` channel with a delay of 10 seconds.  
+
+**Disable slowmode:**
+```
+?slowmode #bot-usage 0
+```
+will disable slowmode on the `#bot-usage` channel.  
+    ",
+        command = "?slowmode {channel} {seconds}",
```
Comment (tinaun) [inline src/api.rs:97]: should this just be a static string?

**Review by tinaun — APPROVED** [review]
lgtm after a rebase

File: src/api.rs (hunk 1 above)
Comment (technetos) [reply src/api.rs]: Nice catch!

**Pull request merged**